test/file_acceptor/evidence_uploader.js

```diff
@@ -0,0 +1,4 @@
+const { Logger } = require('@hmcts/nodejs-logging');
+const { bootstrap } = require('test/file_acceptor');
+
+bootstrap(() => Logger.log('Started file acceptor'));
```

Comment:
What's this part for?

Reply:
This allows us to run the file-acceptor locally